frontend/config.json

```diff
@@ -1,4 +1,5 @@
 {
   "GOOGLE_CLIENT_ID": "518450911507-fu0js4mqkdhbsspfqcmtkddn76j7frns.apps.googleusercontent.com",
-  "GOOGLE_CLIENT_SECRET": "vsgI6I-wm_ICXWBhVmkGVa9f"
+  "GOOGLE_CLIENT_SECRET": "vsgI6I-wm_ICXWBhVmkGVa9f",
+  "BACKEND_URL": "http://localhost:8000/api"
```

Comment:
Do you mind storing this info in a different file? I was planning to clean this one out of the repo and to add it to .gitignore since it has secret info inside

Reply:
We don't need the data, I generate production version of it during the build:
`echo '{"GOOGLE_CLIENT_ID":"${GOOGLE_CLIENT_ID}","GOOGLE_CLIENT_SECRET":"${GOOGLE_CLIENT_SECRET}","BACKEND_URL":"https://${DOMAIN_NAME}/api"}' > ./frontend/config.json`

Bud we do need the file, with keys in it or the frontend won't start in dev environment.
I can replace credentials with dome dummy strings. It'll break google login in dev environment, but we know it's working just fine, so that is acceptable.